requirements.txt

```diff
@@ -2,8 +2,8 @@
 v1pysdk-unofficial==0.4.post4
 
 helga>=1.6.0
-Twisted==13.1.0
-pymongo==2.5.2
-httplib2==0.9.2
-oauth2client==2.0.1
-expiringdict==1.1.3
+Twisted>=13.1.0
+pymongo>=2.5.2
+httplib2>=0.9.2
+oauth2client>=2.0.1
+expiringdict>=1.1.3
```

Comment:
I don't like unpinned versions in requirements.txt if there's updated versions you've tested with feel free to bump them up, but we don't want the automated deploy stuff pulling in future versions that aren't compatible.